WordPress/src/main/java/org/wordpress/android/ui/reader/discover/viewholders/ReaderPostViewHolder.kt

```diff
@@ -65,7 +65,6 @@ class ReaderPostViewHolder(
         updateFeaturedImage(state)
         uiHelpers.updateVisibility(image_video_overlay, state.videoOverlayVisibility)
         uiHelpers.setTextOrHide(text_photo_title, state.photoTitle)
-        uiHelpers.updateVisibility(frame_photo, state.photoFrameVisibility)
```

Comment:
Shouldn't we just replace `frame_photo` with `image_featured` on this line instead of removing it?

Reply:
Yep but it should also be visible when featured video is available - it is noticeable after removing the "Group" element. Updated here: 38bd162